### ISSUE TITLE
textbox: add blue focus indicator

### ISSUE THREAD

**Issue by ArtBlue** [eBay/skin#2089]
Currently, using `textbox` with other form controls creates an inconsistent focus experience - textbox does not have the blue focus outline like other controls. 

For this issue, add the blue focus indicator like the one on `select` for form control focus consistency.

-----

![image](https://github.com/eBay/skin/assets/1675667/f0c5e996-aecd-4d38-af11-e6d9871ce44d)

**Comment by agliga** [comment]
Is this as per design? I know we change the border color when we focus on textbox. We should be consistent but also check with design. 
We have it explicity set as 0 in the code https://github.com/eBay/skin/blob/master/src/less/textbox/textbox.less#L135
It could be legacy but it could be intentional (its hard to tell from the history).
